### ISSUE TITLE
Changes required on FOOTER CSS

### ISSUE THREAD

**Issue by lovelymahor** [anuragverma108/SwapReads#2053]
### Is there an existing issue for this?

[X] I have searched for existing issues

### What happened?

In light mode, the current color contrast between the text and background on the website does not meet accessibility standards, making it difficult for users with visual impairments to read the content clearly.
The text should have a high contrast ratio against the background, making it easily readable.
The text blends into the background, making it difficult to read.

![Screenshot (144)](https://github.com/anuragverma108/SwapReads/assets/141830010/ca91ba41-8749-42ce-a9e9-6a7d8f84570e)

Please assign it to me I want to work on this issue. Also, please add the label GSSOC and the level that you find appropriate for this issue. thank you!

### Record

- [X] I agree to follow this project's Code of Conduct
[X] I'm a GSSOC'24 contributor
- [X] I want to work on this issue

**Comment by meenakshiiyer2531** [comment]
Hello @anuragverma108 ,
Can this issue be assigned to me for Gssoc'24. 
Regards,
V Meenakshi Iyer
